### PR TITLE
Update linux on Travis to Ubuntu Xenial 16.04

### PR DESCRIPTION
**Pull request by xnull**
## Overview

Description:
Update Linux distribution from ubuntu trusty(14) to ubuntu xenial (16.04)

Why should this be merged: 
Ubuntu trusty comes too old

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: java
 sudo: required
-dist: trusty
+dist: xenial
```
Comment (StrikingThirteen) [inline .travis.yml]: Why 16.04 and not 18.04?

Comment (xnull) [reply .travis.yml]: Latest version Travis can provide https://docs.travis-ci.com/user/reference/overview/

**Comment by StrikingThirteen** [comment]
Lgtm

**Comment by xnull** [comment]
Can't build projects with xenial, oraclejdk8 doesn't work. See https://travis-ci.community/t/the-travis-ci-build-could-not-be-completed-due-to-an-error/1345

**Comment by StrikingThirteen** [comment]
seems we are stuck with trusty :)

**Pull request closed without merging**